pyproject.toml

```diff
@@ -0,0 +1,2 @@
+[build-system]
+requires = ["cython", "numpy", "setuptools", "wheel"]
```

Comment:
can you also explicitly specify the `build-backend` to 

> build-backend = "setuptools.build_meta"

It looks like the best practice. Not sure if it actually is. Maybe we're just being explicit here. Ref https://snarky.ca/what-the-heck-is-pyproject-toml/

Reply:
That makes sense, I saw that as well, but also was following https://github.com/enthought/scimath/blob/master/pyproject.toml

I will add that now, and possibly a PR in scimath doing the same